python/lsst/fgcmcal/fgcmBuildStars.py

```diff
@@ -626,19 +654,41 @@ def fgcmMakeAllStarObservations(self, groupedDataRefs, visitCat):
 
                 sources = dataRef.get(datasetType='src', flags=afwTable.SOURCE_IO_NO_FOOTPRINTS)
 
+                # If we are subtracting the local background, then correct here
+                # before we do the s/n selection.  This ensures we do not have
+                # bad stars after local background subtraction.
+
+                if self.config.doSubtractLocalBackground:
+                    # At the moment we only adjust the flux and not the flux
+                    # error by the background because the error on
+                    # base_LocalBackground_instFlux is the rms error in the
+                    # background annulus, not the error on the mean in the
+                    # background estimate (which is much smaller, by sqrt(n)
+                    # pixels used to estimate the background, which we do not
+                    # have access to in this task).  In the default settings,
```

Comment:
It seems dangerous/unfortunate to pay no heed to the errors.  Perhaps we should have a "numberOfPixelsUsed" column added to the LocalBackground schema?

Reply:
That could be useful, but not necessary.  The errors are truly less than a millimag, but this is something to consider.